<launch>
	<node name="stage" pkg="stage_ros" type="stageros"
	      args="$(find lab1)/world/pursuer.world"/>
	<node name="mover" pkg="lab1" type="evader.py" ns="robot_0"/>
	<node name="tf_broadcaster" pkg="lab1" type="tf_broadcaster.py" ns="robot_0" />
	<node name="tf_broadcaster" pkg="lab1" type="tf_broadcaster.py" ns="robot_1" />
	<node name="tf_listener" pkg="lab1" type="pursuer-evader.py" ns="robot_1" />		     
</launch>
